### ISSUE TITLE
Open Folder Select Screen in User Folder

### ISSUE THREAD

**Issue by adcrn** [adcrn/knest#12]
A suggestion from @leavens:
>It would be useful to be able to start in the user's Pictures directory; instead of having that be found automatically (which might be hard to make portable), you could let the user set a starting directory in settings.

**Comment by adcrn** [comment]
This has been implemented in https://github.com/adcrn/knest/commit/7bb1a2b934dd626d14549657fcaa3228dd5256e2.

From `config.kv`:
```
<LoadDialog>:
    list: copy.copy(gv.dir_paths)
    cwd: os.getcwd()
    fancy_path: Path(root.cwd)
    start_path: os.path.join(root.fancy_path.parts[0], root.fancy_path.parts[1], root.fancy_path.parts[2])
```
...
```
FileChooserIconView:
    id: filechooser
    dirselect: True
    rootpath: root.start_path
```

It works under the assumption that modern operating systems (e.g. macOS, Linux distros, Windows 8/10) have a root folder structure that typically has the form `/Users/blah`, `/home/blah`, or `C:/Users/blah`, respectively. The application will now open in the root folder of the user, where people will typically store their images.

**Comment by leavens** [comment]
Hi Alexander,


Great, thanks!

        Regards,

        Gary T. Leavens
        437D Harris Center (Bldg. 116)
        Computer Science, University of Central Florida
        4000 Central Florida Blvd., Orlando, FL 32816-2362 USA
        http://www.cs.ucf.edu/~leavens  phone: +1-407-823-4758
        leavens@cs.ucf.edu

**Comment by leavens** [comment]
Hi Alexander,

On windows, when I start this version from cygwin, I'm trapped under
cygwin's root, which is c:/cygwin64 for me. But my pictures directory is in
c:/Users/leave/Pictures, which I can't get to from c:/cygwin64 so this
doesn't work for me.   This happens even if I start from a Windows cmd
shell (not cygwin).

        Regards,

        Gary T. Leavens
        437D Harris Center (Bldg. 116)
        Computer Science, University of Central Florida
        4000 Central Florida Blvd., Orlando, FL 32816-2362 USA
        http://www.cs.ucf.edu/~leavens  phone: +1-407-823-4758
        leavens@cs.ucf.edu

**Comment by adcrn** [comment]
@leavens Can you pull the latest master commit and try again under both Cygwin and the Windows command shell?

**Comment by leavens** [comment]
Hi Alexander,

Yes. On cygwin I'm still trapped below my home directory
(c:/cygwin64/home/leavens), and there is no ".." directory in the folder
selection to get out. The same thing happens with a windows cmd prompt,
probably due to some environment variables I have set to tell shells where
my home directory is.  So the new change doesn't help.

I suggest that you not use something that changes the root of the process
to a directory below the machine's root. If the process can start in a
directory without making that directory the root directory for the process
(in the sense of chroot) then that will be more flexibile.

        Regards,

        Gary T. Leavens
        437D Harris Center (Bldg. 116)
        Computer Science, University of Central Florida
        4000 Central Florida Blvd., Orlando, FL 32816-2362 USA
        http://www.cs.ucf.edu/~leavens  phone: +1-407-823-4758
        leavens@cs.ucf.edu

**Comment by adcrn** [comment]
>It would be useful to be able to start in the user's Pictures directory; **instead of having that be found automatically (which might be hard to make portable), you could let the user set a starting directory in settings.**

This will be implemented in #19. Closing this issue for now.
